### PR TITLE
Watsonx documentation update

### PR DESCRIPTION
**Pull request by mdswift**
- Add links to watsonx use cases: complex tasks, conversational search, summarization
- Add details on how to find the watsonx project id

### CODE REVIEW

File: integrations/extensions/starter-kits/language-model-watsonx/README.md [hunk 1]
```diff
@@ -2,23 +2,26 @@
...
-1. To find your watsonx project ID, go to [your projects](https://dataplatform.cloud.ibm.com/projects/). Open the project you would like to work in, and go to the Manage tab and the Services and integrations tab. Click Associate service + and associate an instance of Watson Machine Learning (WML).
-   If you do not have an instance of WML, you can provision a Lite instance from the [IBM Cloud catalog](https://cloud.ibm.com/catalog/services/watson-machine-learning). Then complete this step using the new Lite instance.
+1. To find your watsonx project id, go to [watsonx.ai](https://dataplatform.test.cloud.ibm.com/wx) and find Projects/<project-name> (this could be your `sandbox`, which is created for you by default). Click on the <project-name> link, then follow the Project's Manage tab (Project -> Manage -> General -> Details) to find the project id.
 
-### Create an assistant
+### Create an assistant◊
```
Comment (jwm4) [inline integrations/extensions/starter-kits/language-model-watsonx/README.md:24]: Why is this ◊ here?  Is it a typo?

File: integrations/extensions/starter-kits/language-model-watsonx/README.md [hunk 2]
```diff
@@ -2,23 +2,26 @@
 
 watsonx is our enterprise-ready AI and data platform designed to multiply the impact of AI across your business. It provides an API for interacting with generative language models. You can use this starter kit to connect IBM Watson Assistant to the watsonx API.
 
-This starter kit does not provide examples of anything that you can _do_ with this extension. It can be used for specific use cases as the generative language model component.
+## Example use cases with the watsonx custom extension
+
+- [Complex informational tasks: Use watsonx to generate natural language answers for specific topics with only a small amount of relevant text](../language-model-complex-informational-tasks/README.md)
```
Comment (jwm4) [inline integrations/extensions/starter-kits/language-model-watsonx/README.md:7]: The watsonx kit for this one is not merged yet.  Should we wait to merge this change until that one is merged too?

**Review by jwm4 — CHANGES_REQUESTED** [review]

File: integrations/extensions/starter-kits/language-model-watsonx/README.md (hunk 2 above)
Comment (mdswift) [reply integrations/extensions/starter-kits/language-model-watsonx/README.md:7]: Yes, it makes sense to wait.

File: integrations/extensions/starter-kits/language-model-watsonx/README.md (hunk 1 above)
Comment (mdswift) [reply integrations/extensions/starter-kits/language-model-watsonx/README.md]: Yes, a typo. Fixed.

**Review by jwm4 — APPROVED** [review]
This was blocked on the watsonx complex info kit being merged.  Now that one is merged, so it is time to merge these documentation updates too.

**Pull request merged**